addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -13,8 +15,11 @@ const setCursor = OdooEditorLib.setCursor;
 const LinkTools = Link.extend({
     template: 'wysiwyg.widgets.linkTools',
     events: _.extend({}, Link.prototype.events, {
-        'click we-select we-button': '_onPickSelectOption',
+        'click we-select :not([name="link_border_style"]) we-button': '_onPickSelectOption',
```

Comment:
In fact this change just does nothing I think? All button in selects are descendant of `:not(anything)`... so here my guess is that this handler is still called for the case you somehow did not want it to...

But from what I see, this is a sad handler anyway, seems to do nothing but force active on the button that is clicked whether or not the button did do something or not...